Modified IoT Hub "Get Started" samples for simulated devices

Ilias-Tsigkogiannis opened the pull request:
Modified packet to include temperature and humidity data, in order to make the samples compatible with the other "Get Started" tutorials

Ilias-Tsigkogiannis committed:
Updated IoT Hub "Get Started" tutorial for node

Added temperature and humidity to packet, in order to make it compatible with the other "Get Started" tutorials for IoT Hub

Ilias-Tsigkogiannis committed:
Updated IoT Hub "Get Started" sample for Java

Added temperature and humidity to packet structure, in order to make it compatible with the other "Get Started" tutorials

Ilias-Tsigkogiannis committed:
Updated description for MessageSender class

Ilias-Tsigkogiannis committed:
Updated IoT Hub "Get Started" sample for .NET

Added temperature and humidity fields in the packet structure, in order to make it compatible with the other "Get Started" tutorials

dominicbetts commented on articles/iot-hub/iot-hub-csharp-csharp-getstarted.md:
```diff
@@ -196,7 +203,7 @@ In this section, you create a .NET console app that simulates a device that send
         }
    
-    This method sends a new device-to-cloud message every second. The message contains a JSON-serialized object, with the device ID and a randomly generated number to simulate a wind speed sensor.
+    This method sends a new device-to-cloud message every second. The message contains a JSON-serialized object, with the device ID and randomly generated numbers to simulate a wind speed sensor, a temperature sensor and a humidity sensor.
```
Minor tweak - missing comma. Should be: "This method sends a new device-to-cloud message every second. The message contains a JSON-serialized object, with the device ID and randomly generated numbers to simulate a wind speed sensor, a temperature sensor, and a humidity sensor. "

dominicbetts commented on articles/iot-hub/iot-hub-java-java-getstarted.md:
````diff
@@ -392,7 +401,7 @@ In this section, you create a Java console app that simulates a device that send
     ```
     
-    This method sends a new device-to-cloud message one second after the IoT hub acknowledges the previous message. The message contains a JSON-serialized object with the deviceId and a randomly generated number to simulate a wind speed sensor.
+    This method sends a new device-to-cloud message one second after the IoT hub acknowledges the previous message. The message contains a JSON-serialized object with the deviceId and randomly generated numbers to simulate a wind speed sensor, a temperature sensor and a humidity sensor.
````
Minor tweak - missing comma. Should be: "This method sends a new device-to-cloud message one second after the IoT hub acknowledges the previous message. The message contains a JSON-serialized object with the deviceId and randomly generated numbers to simulate a wind speed sensor, a temperature sensor, and a humidity sensor."

dominicbetts commented on articles/iot-hub/iot-hub-java-java-getstarted.md:
```diff
@@ -364,16 +366,23 @@ In this section, you create a Java console app that simulates a device that send
     
             String msgStr = telemetryDataPoint.serialize();
             Message msg = new Message(msgStr);
+            msg.setProperty("temperatureAlert", (currentTemperature > 28) ? "true" : "false");
```
When I try this code, I get an error when the message with the property is sent. The EventCallback method shows:
`IoT Hub responded to message with status: ERROR`

I'm using version 1.1.23 of the iot-device-client Maven package.

Ilias-Tsigkogiannis committed:
Removed windSpeed from sample

Removed windSpeed from sample, in order to make it consistent with the samples for physical devices in docs.microsoft.com

Ilias-Tsigkogiannis committed:
Updated sample for Java

Removed windSpeed, in order to make sample consistent with the "Get Started" samples for physical devices in docs.microsoft.com, and fixed error in sample.

Ilias-Tsigkogiannis committed:
Updated threshold for temperatureAlert

Ilias-Tsigkogiannis committed:
Updated "Get Started" sample for C#

Updated threshold for temperatureAlert and fixed typo
